### PR TITLE
[Merged by Bors] - chore: adaptations to nightly-2024-03-05

### PR DESCRIPTION
**Pull request by kim-em**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Tactic/FieldSimp.lean [hunk 1]
```diff
@@ -155,6 +155,7 @@ elab_rules : tactic
   -- Prior to https://github.com/leanprover/lean4/pull/3523,
   -- the maxDischargeDepth wasn't actually being checked: now we have to set it higher.
   let cfg := { cfg with maxDischargeDepth := 7 }
+
   let loc := expandOptLocation (mkOptionalNode loc)
```
Comment (Ruben-VandeVelde) [inline Mathlib/Tactic/FieldSimp.lean:159]: Revert?

File: Mathlib/Data/Bool/Basic.lean [hunk 2]
```diff
@@ -243,8 +208,8 @@ attribute [simp] xor_assoc
 
 #align bool.bxor_ff_left Bool.false_xor
 #align bool.bxor_ff_right Bool.xor_false
-#align bool.band_bxor_distrib_left Bool.and_xor_distrib_left
-#align bool.band_bxor_distrib_right Bool.and_xor_distrib_right
+#noalign bool.band_bxor_distrib_left
+#noalign bool.band_bxor_distrib_right
```
Comment (semorrison) [inline Mathlib/Data/Bool/Basic.lean:212]:
```suggestion
#align bool.band_bxor_distrib_left Bool.and_xor_distrib_left
#align bool.band_bxor_distrib_right Bool.and_xor_distrib_right
```

File: Mathlib/Data/Erased.lean [hunk 3]
```diff
@@ -153,7 +153,7 @@ theorem map_def {α β} : ((· <$> ·) : (α → β) → Erased α → Erased β
   rfl
 #align erased.map_def Erased.map_def
 
--- Porting note: Old proof `by refine' { .. } <;> intros <;> ext <;> simp`
+--Porting note: Old proof `by refine' { .. } <;> intros <;> ext <;> simp`
```
[the remark this answers is not in the document]
Comment (semorrison) [reply Mathlib/Data/Erased.lean:156]:
```suggestion
-- Porting note: Old proof `by refine' { .. } <;> intros <;> ext <;> simp`
```

File: Mathlib/Data/List/Sublists.lean [hunk 4]
```diff
@@ -97,7 +97,8 @@ theorem length_sublists' : ∀ l : List α, length (sublists' l) = 2 ^ length l
   | [] => rfl
   | a :: l => by
     simp_arith only [sublists'_cons, length_append, length_sublists' l,
-      length_map, length, Nat.pow_succ', mul_succ, mul_zero, zero_add, npow_eq_pow, pow_eq]
+      length_map, length, Nat.pow_succ', mul_succ, mul_zero, zero_add]
+    rfl
```
Comment (semorrison) [inline Mathlib/Data/List/Sublists.lean:101]:
```suggestion
      length_map, length, Nat.pow_succ', mul_succ, mul_zero, zero_add, npow_eq_pow, pow_eq]
```

File: Mathlib/Tactic/FieldSimp.lean (hunk 1 above)
Comment (semorrison) [reply Mathlib/Tactic/FieldSimp.lean:159]:
```suggestion
  let cfg := { cfg with maxDischargeDepth := 7 }
  let loc := expandOptLocation (mkOptionalNode loc)
```

**Comment by semorrison** [comment]
Thanks @Ruben-VandeVelde for those catches.

Unfortunately it's a manual process keeping `bump/nightly-XXXX` and `nightly-testing` in sync, and I seem to get it wrong a lot. :-(

**Review by kmill — APPROVED** [review]
The changes look fine to me

**Pull request closed without merging**